api/src/db/DynamoDataClient.ts

```diff
@@ -72,6 +72,16 @@ export const DynamoDataClient = (
     }
   };
 
+  const findUserByBusinessName = async (businessName: string): Promise<UserData[]> => {
+    const usersWithBusinesses = await userDataClient.queryUsersWithBusinesses();
```

Comment:
Added normalization (trimming and lowercasing) of businessName for case-insensitive and whitespace-agnostic comparison. to prevent false mismatches caused by whitespace/case sensitivity.

Reply:
We may want some sort of 'fuzzy' matching, since we may not have the exact business name when looking up businesses.